cnf/repo/org.osgi.impl.service.cm/org.osgi.impl.service.cm-7.0.0.lib

```diff
@@ -1 +1,2 @@
 org.apache.felix.configadmin; version=1.9
+org.osgi.test.junit5.cm; version=latest
```

Comment:
A testing jar must not be in the impl lib file!

Reply:
sure but if i put it into the junit.tck all other test will fail because of the  missing cm and adding it just to the bndrun file just works in ide but not on ci because it does not find it on one of the given repos.

Reply:
There is no reason some other project using the CM impl needs this osgi-test jar. You already have it in the -runbundles and -buildpath of the TCK which should be sufficient. Yes, you cannot put it in the osgi.tck.junit-platform.bnd file.